lib/DDG/Spice/NobelPrize.pm

```diff
@@ -0,0 +1,43 @@
+package DDG::Spice::NobelPrize;
+
+# ABSTRACT: Write an abstract here
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+use Data::Dumper;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'http://api.nobelprize.org/v1/prize.json?' 
+                . '&year=$1' 
+                . '&category=$2';
+
+
+# Triggers - https://duck.co/duckduckhack/spice_triggers
+triggers startend => 'nobel';
+
+spice from => '([^-]+)-([^-]+)';
```

Comment:
These captures need to be optional because one of them can always be empty: `spice from => "([^-]+)?-([^-]+)?";`